### DISCUSSION TITLE
Altri tipi di voti

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#582] [Idee e proposte]
Sarebbe possibile aggiungere i voti tipo allontanamento, ammonizione, ecc.
Allego immagine
![Screenshot_20231025_144321](https://github.com/iisgiua/giuaschool/assets/137722020/e4493b76-7541-402b-a13c-34bab111cea9)

**Comment by trinko** [comment]
No, attualmente non è possibile personalizzare i voti.
Puoi però semplicemente usare il campo GIUDIZIO presente nella valutazione, dove puoi scrivere quello che vuoi.
